src/Microsoft.Data.SqlClient/src/Microsoft/Data/Common/ActivityCorrelator.cs

```diff
@@ -19,16 +19,14 @@ internal sealed class ActivityId
             internal readonly Guid Id;
             internal readonly uint Sequence;
 
-            internal ActivityId(uint sequence)
+            internal ActivityId(Guid? currentActivityId, uint sequence = 1)
```

Comment:
Why add the sequence default value?

Reply:
Should be the default since the sequence starts from 1 (it's set when reading 'ActivityId.Current')